design-editor/src/panel/property/attribute/templates/attribute-background.html

```diff
@@ -1,6 +1,7 @@
 <li>
     <span class="closet-attribute-common-label-long">color</span>
-    <input type="color" name="backgroundColor" class="closet-attribute-common-textbox">
+	<input type="color" name="backgroundColor" class="closet-attribute-common-textbox">
+	<button id="backgroundColorClear" style="display:none;">Reset background color</button>
```

Comment:
Please consider add class `btn` to the button element then it will be styled in our internal styling. And I am not sure if this button text isn't too long. Maybe 'Reset color' would be enough?

Reply:
Sure, I will change button text and add button styling